src/jaxoplanet/light_curves.py

```diff
@@ -66,6 +66,10 @@ def light_curve(
         if texp is None:
             tgrid = t
         else:
+            if (
+                texp.ndim == 0
+            ):  # If a scalr is passed, automatically broadcast to the shape of t
+                texp = jnpu.repeat(texp, len(t))
```

Comment:
I think `jnpu.broadcast_to(texp, jnp.shape(t))` might be better?

Reply:
Okay!